tests/ui/author/blocks.stdout

```diff
@@ -0,0 +1,13 @@
+if_chain! {
+    if let ExprKind::Block(ref block) = expr.node;
+    if let Some(trailing_expr) = &block.expr;
+    if block.stmts.len() == 0;
+    then {
+        // report your lint here
+    }
+}
+if_chain! {
+    then {
```

Comment:
Is this empty `if_chain` intended?

Reply:
yea, it exists for the `foo` function which we don't handle yet. So it's mainly a canary.